Makefile

```diff
@@ -6,20 +6,24 @@ clean:
 	find . -name '*~' -delete
 	find . -name '.coverage.*' -delete
 
-PYTEST_CMD := poetry run pytest -s -vv gridstatus/ -n auto
-NOT_SLOW := -m "not slow" --reruns 5 --reruns-delay 3
+PYTEST_CMD := poetry run pytest -s -vv gridstatus/ -n auto --reruns 5 --reruns-delay 3
+NOT_SLOW_AND_NOT_REQUIRES_ERCOT_AUTH := -m "not slow and not requires_ercot_auth"
```

Comment:
`-m "not slow" -m "not requires_ercot_auth"` means `-m "not slow or not requires_ercot_auth"` but we don't want to run either, so we have to combine them in one expression.